cmd/agent/install_script.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 # (C) Datadog, Inc. 2010-present
 # All rights reserved
-# Licensed under Simplified BSD License (see LICENSE)
+# Licensed under Apache-2.0 License (see LICENSE)
```

Comment:
I suppose this is fine, but I am curious about the rationale (I guess this happened when switching to the other repo)

Reply:
Yeah, the line also says "see LICENSE", but that file contains Apache-2.0 license. Since everything else in the agent is ASL 2.0, we decided to change the line to contain Apache-2.0 as well.

Reply:
Makes sense, thanks!